ansible/action_plugins/agnosticd_user_info.py

```diff
@@ -88,16 +88,19 @@ def run(self, tmp=None, task_vars=None):
                 )
             )
             if not user and msg != None:
+                os.makedirs(os.path.dirname(os.path.join(output_dir, 'user-info.yaml')), exist_ok=True)
```

Comment:
Why join then strip off dir with dirname? why not just?
```
                os.makedirs(output_dir, exist_ok=True)
```

Reply:
Because I'm lazy and totally dumb, so I just copy/pasted the code for the path that did not work from the other line. 😔
I will correct this.

Reply:
Change made, thanks for spotting this Johnathan!